src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How to manage the quality of your alerts with New Relic"
+redirects: 
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide/
+  - /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alert-quality-management-tutorial/
+  - /docs/new-relic-solutions/observability-maturity/aqm-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide
+---
+
+Teams suffer from alert fatigue when experiencing high alert volumes and alerts that aren't aligned to business impact. When this happens,incident responders often assume the alerts are false and have no business impact. In turn, they may start to prioritize easy-to-resolve alerts over others or close unresolved incidents to stay within their response time targets. This almost always results in slower incident response and increased scope and severity when major issues occur.
```

Comment:
> Teams suffer from alert fatigue when experiencing high alert volumes and alerts that aren't aligned to business impact. 

I'd love to see this be more active or address the reader directly. `suffer from... when experiencing... that aren't aligned to` are pretty passive/complex verb statements.